src/main/java/org/entando/plugins/pda/EntandoPdaPluginJavaApplication.java

```diff
@@ -2,27 +2,47 @@
 
 import org.entando.connectionconfigconnector.config.ConnectionConfigConfiguration;
 import org.entando.connectionconfigconnector.service.ConnectionConfigConnector;
+import org.entando.connectionconfigconnector.service.impl.ConnectionConfigConnectorImpl;
 import org.entando.connectionconfigconnector.service.impl.InMemoryConnectionConfigConnector;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Import;
 import org.springframework.context.annotation.Profile;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 @SpringBootApplication
 @ComponentScan("org.entando")
 @Import(ConnectionConfigConfiguration.class)
 @SuppressWarnings("PMD.UseUtilityClass")
 public class EntandoPdaPluginJavaApplication {
 
+    @Value("${pda.mock-connection-config}")
+    private boolean mockConnectionConfig;
+
     public static void main(final String[] args) {
         SpringApplication.run(EntandoPdaPluginJavaApplication.class, args);
     }
 
+    @Bean
+    public ConnectionConfigConnector connectionConfigConnector(ConnectionConfigConnectorImpl connectionConfigImpl) {
+        if (mockConnectionConfig) {
+            return new InMemoryConnectionConfigConnector();
+        }
+        return connectionConfigImpl;
+    }
+
     @Bean
     @Profile("dev")
-    public ConnectionConfigConnector connectionConfigConnector() {
-        return new InMemoryConnectionConfigConnector();
+    public WebMvcConfigurer corsConfigurer() {
+        return new WebMvcConfigurer() {
+            @Override
+            public void addCorsMappings(CorsRegistry registry) {
+                registry.addMapping("/*").allowedOrigins("http://localhost:3333");
```

Comment:
Shouldn't this localhost url come from the `application.properties` file?

Reply:
I think it would be good because we could overwrite the url. I will handle it in a different PR.